.github/workflows/release.yml

```diff
@@ -12,6 +12,7 @@ on:
 jobs:
   test:
     name: Tests
+    environment: Production
```

Comment:
Should we keep the `environment` approval for the `release` job? In this way, tests always run when you open a PR and get feedback straight away, but we require approval for the `release` one?